OSDOCS-8888 z-stream RNs for 4.12.45 (#68930)
openshift/openshift-docs: release_notes/ocp-4-12-release-notes.adoc

[id="ocp-4-12-44-updating"]
==== Updating

To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

[id="ocp-4-12-45"]
=== RHSA-2023:7608 - {product-title} 4.12.45 bug fix and security update

Issued: 2023-12-06

{product-title} release 4.12.45, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7608[RHSA-2023:7608] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7610[RHSA-2023:7610] advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.12.45 --pullspecs
----

[id="ocp-4-12-45-bug-fixes"]
==== Bug fixes

> * Previously, using Cluster Autoscaler with Nodes that have CSI storage would cause the Cluster Autoscaler to enter a `CrashBackoff` loop. With this release, dependencies have been updated to check the nil pointer, and the bug is no longer present. (link:https://issues.redhat.com/browse/OCPBUGS-23274[*OCPBUGS-23274*])

Reviewer: > AFAIK, we use the `CrashLoopBackoff` term often in the docs hence I suggest to consider using "`CrashLoopBackoff` state".

+1 for using the term `CrashLoopBackoff`.  A minor nit based on its usage across the docs - "`CrashLoopBackoff` status".

> * Previously, using Cluster Autoscaler with Nodes that have CSI storage would cause the Cluster Autoscaler to enter a `CrashBackoff` loop. With this release, dependencies have been updated to check the nil pointer, and the bug is no longer present. (link:https://issues.redhat.com/browse/OCPBUGS-23274[*OCPBUGS-23274*])

Author: Thank you!

> * Previously, EgressIP could not be applied to the Egress node on an Azure private cluster. A patch has been created to work around this issue. With this release, EgressIP is enabled for Azure setups that use outbound rules to achieve outbound connectivity. An architectural constraint in Azure prevents the secondary IP (acting as EgressIP) from having outbound connectivity in such setups. This means that matching pods will have no outbound connectivity to the internet, but will be able to reach external servers in the infrastructure network, which is the intended use case for EgressIP. (link:https://issues.redhat.com/browse/OCPBUGS-22949[*OCPBUGS-22949*])

Reviewer 2: 1. I am wondering if explaining the use case here is required?
2. "Azure setups" mean "Azure private clusters" here right?

> * Previously, there was no suitable virtual media device for Cisco UCS Blade. With this release, Cisco UCS hardware can be provisioned using Redfish virtual media. (link:https://issues.redhat.com/browse/OCPBUGS-19064[*OCPBUGS-19064*])

Reviewer: Minor edit to Servesha's suggestion to avoid passive tone.
```suggestion
* Previously, there was no suitable virtual media device for Cisco UCS Blade. With this release, you can use  Redfish virtual media to provision Cisco UCS hardware. (link:https://issues.redhat.com/browse/OCPBUGS-19064[*OCPBUGS-19064*])
```
